docs/docsite/examples/lists_mergeby/example-001.out

```diff
@@ -0,0 +1,10 @@
+  list3:
```

Comment:
Files in this directory will not be found during the docs build. The files must be in docs/docsite/rst/.

Reply:
I'll move it inside the 'rst' directory. Thank you.

Reply:
I'll try to build the docs to see whether it actually works.

Reply:
> I'll try to build the docs to see whether it actually works.

JFYI. I was running sphinx-view while editing the docs.

Reply:
Unfortunately it does not work:
```
docsite/rst/collections/community/general/docsite/filter_guide.rst:257: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-001.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:262: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-001.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:269: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-002.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:274: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-002.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:289: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-003.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:294: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-003.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:299: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-004.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:304: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-004.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:309: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-005.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:314: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-005.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:319: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-006.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:324: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-006.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:329: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-007.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:334: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-007.out' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:339: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-008.yml' not found or reading it failed
docsite/rst/collections/community/general/docsite/filter_guide.rst:344: WARNING: Include file 'docsite/rst/collections/community/general/docsite/examples/lists_mergeby/example-008.out' not found or reading it failed
```
Antsibull only copies `.rst` files over (https://github.com/ansible-community/antsibull/blob/1eb595309ec0e01310edd59682e808dde3fc8a3a/src/antsibull/extra_docs.py#L44-L60, https://github.com/ansible-community/antsibull/blob/1eb595309ec0e01310edd59682e808dde3fc8a3a/src/antsibull/extra_docs.py#L194-L206). So for now you have to inline the files instead of using `literalinclude`. (I would still keep the files, so that it's easier to reproduce and to eventually replace the inlined copies with the originals.)

Reply:
This is not only awkward but error-prone as well. Would it be possible to try the *.rst files I submitted?